main.py

```diff
@@ -48,6 +51,14 @@ def check_start_idx(sections):
             errcount += 1
     return errcount
 
+def check_word_counts(sections, transcript):
```

Comment:
Can this PR check something more directly based on what dsRAG returns? Such as bogus segment offsets